server/src/test/java/keywhiz/IntegrationTestRule.java

```diff
@@ -29,4 +29,11 @@ public static RuleChain rule() {
         .outerRule(new MigrationsRule())
         .around(new DropwizardAppRule<>(KeywhizService.class, configPath));
   }
+
+  public static RuleChain rule(String configFile) {
```

Comment:
overload constructor so that I can specify different config files for testing `createSecret` in SecretResource